vllm_ascend/patch/platform/patch_common/__init__.py

```diff
@@ -15,4 +15,6 @@
 # limitations under the License.
 #
 
+import vllm_ascend.patch.platform.patch_common.patch_arg_utils
+import vllm_ascend.patch.platform.patch_common.patch_config
```

Comment:
there is a typo in the naming of file "pacth_config.py", should be "patch_config.py"